torch/nn/functional.py

```diff
@@ -1014,6 +1014,24 @@ def dropout3d(input, p=0.5, training=True, inplace=False):
 
 def feature_alpha_dropout(input, p=0.5, training=False, inplace=False):
     # type: (Tensor, float, bool, bool) -> Tensor
+    r"""
+    Randomly masks some of the elements of the input tensor. Instead of setting 
```

Comment:
Mention that it wipes out entire channels?

Reply:
Addressed in next update.